frontend/styles/var/forms.scss

```diff
@@ -0,0 +1 @@
+$form-field-label-min-height: 23.6333px;
```

Comment:
Nice use of variables, maybe 24px for ease?

Reply:
Agreed, let's go with `24px`.

Reply:
roger